build.assets/Makefile

```diff
@@ -28,6 +28,8 @@ LOGGING_APP_TAG ?= 0.0.1
 LOGGING_APP_BRANCH ?= $(LOGGING_APP_TAG)
 MONITORING_APP_TAG ?= 0.0.1
 MONITORING_APP_BRANCH ?= $(MONITORING_APP_TAG)
+INGRESS_APP_TAG ?= 0.0.1-dev
```

Comment:
Still says "-dev" here.

Reply:
why are we duplicating those TAGs? that seems dangerous and error prone. 
Anyway fixed it now :+1: